### DISCUSSION TITLE
Fatal: Could not read from remote repository

### DISCUSSION THREAD

**Discussion by ArchLuke** [community/community#22115] [New to GitHub]
<p>My project has had a ton of commits, but sudo git push has suddenly stopped working today… It yields the error</p>
<pre><code class="lang-auto">fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

</code></pre>
<p>I looked around, and the suggestions are all ssh related. The project, however, uses https.  Here is the .git/config file</p>
<pre><code class="lang-auto">[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
[remote "origin"]
	url = https://github.com/ArchLuke/YouSuck.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = origin
	merge = refs/heads/master

</code></pre>
<p>Cloning, pulling, etc all work fine, but pushing isn’t…</p>

**Comment by danielelkington** [comment]
<p>Same with us - everyone in our team suddenly cannot push to one of our repositories with the same error message (but only one of our repositories). Yesterday it was working fine.</p>

**Comment by danielelkington** [comment, marked as answer]
<p>This has started working again.</p>
